src/Controllers/MovieController.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.EntityFrameworkCore;
+using TrainingNet.Models.DataBase;
+using TrainingNet.Repositories;
+using TrainingNet.Repositories.Database;
+using TrainingNet.Repositories.Interfaces;
+
+namespace TrainingNet.Controllers
+{
+    [Route("[controller]")]
+    public class MovieController : Controller
+    {
+        private readonly IHtmlLocalizer<HomeController> _localizer;
+        private readonly IUnitOfWork _unitOfWork;
+        public MovieController(IHtmlLocalizer<HomeController> localizer, IUnitOfWork unitOfWork){
```

Comment:
add an empty line before method declaration. also, by convention, we use the bracket in the next line